server/src/main/java/org/opensearch/index/shard/RemoveCorruptedShardDataCommand.java

```diff
@@ -191,6 +191,11 @@ protected void findAndProcessShardPath(
         }
 
         final IndexSettings indexSettings = new IndexSettings(indexMetadata, settings);
+        if (indexSettings.isRemoteTranslogStoreEnabled()) {
+            // ToDo : Need to revisit corrupt shard recovery strategy for remote store enabled indices
+            throw new OpenSearchException("tool doesn't work for remote translog enabled indices");
```

Comment:
What does this tool do?

Reply:
Based on my quick glance, this is used to clean up corrupted segment files and translog files as well. However we will need to revisit this for remote segment and translog store. This assumes the presence of all files locally and won't work as is for our use case.

Reply:
But in case of remote segment store as well as remote translog, the files are present in local too, right? IMO, the tool should still be able to work.

Reply:
lets add a follow up issue for evaluating this thoroughly.